overrides/config/ftbquests/quests/chapters/create.snbt

```diff
@@ -1,30 +1,3261 @@
+					type: "item"
+				}
+			]
+			title: "Cardboard Armor"
+			x: -5.0d
+			y: -5.0d
+		}
+		{
+			dependencies: ["03900C2AAAAD88AC"]
+			description: ["&ePackagers&r will package the contents of a container when a &cRedstone Signal&r is applied."]
+			id: "5A72D1EA6448B08D"
+			rewards: [{
+				amount: 50L
+				currency: "eternalcurrencies:coins"
+				id: "73377C396D28332E"
+				ignore_reward_blocking: true
+				type: "eternalcurrencies:currency"
+			}]
+			shape: "square"
+			subtitle: "Craft a &ePackager&7 and Package Your First Item"
+			tasks: [
+				{
+					id: "64F010476653AA9C"
+					item: "create:packager"
+					type: "item"
+				}
+				{
+					advancement: "create:packager"
+					criterion: ""
+					id: "763EE2DCDC7D092B"
+					type: "advancement"
+				}
+			]
+			title: "Packager"
+			x: -5.0d
+			y: -2.0d
+		}
+		{
+			dependencies: ["5A72D1EA6448B08D"]
+			description: [
+				"Placing a &eStock Link&f on a &ePackager&f will create your &dLogistics Network&f."
+				""
+				"&cRedstone Signals&r disable the &eStock Link&r."
+				""
+				"&eTip:&f Packages must be transported, but can be requested from anywhere."
+			]
+			icon: "create:stock_link"
+			id: "420A35FB6606D5A6"
+			rewards: [{
+				amount: 100L
+				currency: "eternalcurrencies:coins"
+				id: "5A387FE7E83A6F57"
+				ignore_reward_blocking: true
+				type: "eternalcurrencies:currency"
+			}]
+			shape: "square"
+			subtitle: "Create your first &dLogistics Network"
+			tasks: [
+				{
+					id: "57E54882C7B24017"
+					item: "create:stock_link"
+					type: "item"
+				}
+				{
+					id: "20883FA03265A6C0"
+					item: "create:stock_ticker"
+					type: "item"
+				}
+			]
+			title: "Stock Link"
+			x: -5.0d
+			y: -3.0d
+		}
+		{
+			description: ["Make sure to refuel your &eBacktank&r with rotational power!"]
+			icon: {
+				Count: 1
+				id: "create:netherite_diving_helmet"
+				tag: {
+					Damage: 0
+				}
+			}
+			id: "07BAA8A3B892B3E3"
+			rewards: [
+				{
+					amount: 175L
+					currency: "eternalcurrencies:coins"
+					id: "64E4AA16BBE2ED46"
+					ignore_reward_blocking: true
+					type: "eternalcurrencies:currency"
+				}
+				{
+					id: "31917F45A1004DE0"
+					item: {
+						Count: 1
+						id: "ftbquests:lootcrate"
+						tag: {
+							type: "blackmarket_cache"
+						}
+					}
+					type: "item"
+				}
+			]
+			size: 0.8d
+			subtitle: "Dive Deep Below..."
+			tasks: [
+				{
+					id: "197DADCC31E966B9"
+					item: {
+						Count: 1
+						id: "create:netherite_diving_helmet"
+						tag: {
+							Damage: 0
+						}
+					}
+					type: "item"
+				}
+				{
+					id: "6A1A7BA1FFD20120"
+					item: {
+						Count: 1
+						id: "create:netherite_diving_boots"
+						tag: {
+							Damage: 0
+						}
+					}
+					type: "item"
+				}
+				{
+					id: "46DF5C4610F3C7E2"
+					item: {
+						Count: 1
+						id: "create:netherite_backtank"
+						tag: {
+							Air: 900
+						}
+					}
+					type: "item"
+				}
+				{
+					biome: "alexscaves:abyssal_chasm"

```

Comment:
requiring this biome is an interesting choice. i'd think it would also maybe be good to have a tip about the lava diving feature.